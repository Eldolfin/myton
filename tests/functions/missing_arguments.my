def f(a,b):
  pass
f(1)
