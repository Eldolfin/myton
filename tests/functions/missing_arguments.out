[31merror[error]: Expected 2 arguments but got 1
----- Traceback -----
  â”Œâ”€ <unknown>:3:4
  |   pass
3 | f(1)
  |     ^
[0m